src/dataflow/src/render/reduce.rs

```diff
@@ -462,15 +621,15 @@ where
 // implementation.
 fn build_hierarchical<G>(
     collection: Collection<G, (Row, Row)>,
-    aggrs: Vec<(usize, AggregateExpr)>,
+    expr: RegularHierarchicalReduceExpr,
```

Comment:
i think theres a better way to destructure in function args but i forget the syntax